sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/SelectAsQuery.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Azure.DigitalTwins.Core.QueryBuilder
+{
+    /// <summary>
+    /// Query object either without and clauses or with nonaliased SELECT clause components.
```

Comment:
this needs rewording, 
also, non-aliased should be hyphened I think.

Reply:
`Query that contains aliased selectable properties, optionally preceded by non-aliased ones.`?

Reply:
`Used to select properties with the aliasing mechanism that allows renaming properties when the service responds.` or something along that line.